playground/Client/src/app/analytics/services/google-analytics.service.ts

```diff
@@ -67,4 +67,19 @@ export class GoogleAnalyticsService extends BaseService {
       user_id: hashedUserId, // Use the hashed version of the user ID
     });
   }
+
+  async trackUserGroup(group: "BaseGroup" | "ControlGroup") {
```

Comment:
By A/B test definition, Control Group is the one experiencing already established behavior, i.e., sees no change. Treatment Group is the one receiving new experience. This should be taken into account when pushing analytics data.

Reply:
That's changes from Steve in order to make it be available to the Container App before the meeting today due to his branch code is quite complex at that time. so just ignoring it